get-secretmanager-secrets/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "get-secretmanager-secrets",
+  "version": "0.1.0",
+  "description": "Get Secret Manager secrets GitHub action",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "ncc build src/main.ts",
+    "test": "mocha -r ts-node/register 'tests/*.spec.ts'"
```

Comment:
Please include scripts for format and format-check using prettier, see: https://github.com/GoogleCloudPlatform/github-actions/blob/9ef0382a75df2efcc72850f4e963927a45d59414/setup-gcloud/package.json#L11
Also, please include a precommit script: https://github.com/GoogleCloudPlatform/github-actions/blob/9ef0382a75df2efcc72850f4e963927a45d59414/setup-gcloud/package.json#L12

Reply:
1. What is the value in having `format` and `format-check` when the `eslint` command automatically runs prettifier and fixes any issues?

1. Adding a precommit hook like that doesn't actually work and isn't working in setup-gcloud either. Husky requires a new top-level config and doesn't live under `scripts` anymore ([ref](https://github.com/typicode/husky#upgrading-from-014)). I'm happy to fix it, but can you help me understand what problem it's solving?

Reply:
I added husky with the new config. Still interested in your feedback on 1.